iocore/net/SSLNetProcessor.cc

```diff
@@ -76,6 +77,7 @@ SSLNetProcessor::start(int, size_t stacksize)
     SSLError("Can't initialize the SSL client, HTTPS in remap rules will not function");
   }
 
+  params->InsertCTX(params->clientCertPath, client_ctx);
```

Comment:
And maybe the InsertCTX should be moved to the logic that initializes/sets up the SSLConfig?